webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -92,6 +96,7 @@ export default {
       form: {
         title: '',
         content: '',
+        contentLength: 0,
         teaserImage: null,
         image: null,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T09:14:31Z" title="Thursday, July 18th 2019, 11:14:31 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -177,6 +181,14 @@ export default {
     updateEditorContent(value) {
       // this.form.content = value
```


Remove this comment?